### PR TITLE
Normalising the project

### PR DESCRIPTION
**Pull request by I2evard**
Removed : 
- Default react code (not all).
- Styling applied by default project.

Added :
- context to differentiate between mobile and a browser.
- Base components for NavigationBar and Card
- Normalized CSS file.
- Display flex for all future components.
- Readme files to help with the guidelines on how to structure some files

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -25,3 +25,4 @@
 npm-debug.log*
 yarn-debug.log*
 yarn-error.log*
+backend/.env
```
Comment (uwponcel) [inline .gitignore:28]:
```suggestion

```

Comment (uwponcel) [reply .gitignore:28]: Je l'ai déjà dans le mien ;)

File: frontend/public/index.html [hunk 2]
```diff
@@ -9,6 +9,8 @@
       name="description"
       content="Web site created using create-react-app"
     />
+    <link rel="stylesheet" href="./css/normalize.css" />
+
```
Comment (uwponcel) [inline frontend/public/index.html:13]:
```suggestion

```

File: frontend/src/App.css [hunk 3]
```diff
@@ -40,3 +40,7 @@
     transform: rotate(360deg);
   }
 }
+
+* {
+    display: flex;
```
Comment (uwponcel) [inline frontend/src/App.css:45]: On flex sur tout ?

File: frontend/src/components/NavigationBar/MobileNarBar/MobileNavBar.tsx [hunk 4]
```diff
@@ -0,0 +1,37 @@
...
+    Buttons: styled.button`
+        height: 99%;
+        flex: 1;
+    `
+}
+
+const { Container, Buttons } = StyledMobileNavBar;
+
+export const MobileNavBar = (props: NavBarProps) => {
+
```
Comment (uwponcel) [inline frontend/src/components/NavigationBar/MobileNarBar/MobileNavBar.tsx:29]:
```suggestion

```

File: frontend/src/components/README.md [hunk 5]
```diff
@@ -0,0 +1,12 @@
+# Guide lines for components folder content
+
+Components added to this folder should be the ones that can be reused to build different pages. 
+EX: A generic button, title, icons, etc.
+
+## Naming components
+When naming the components.
```
Comment (uwponcel) [inline frontend/src/components/README.md:7]: Organizing and Naming Components
When organizing and naming components, adhere to the following conventions:

Store components within a directory that carries the general name of the component. For example, if your component is a 'Card', the directory should be named 'Card'.

Example Directory Path: /Card
Inside this general directory, create specific directories for each version of the component tailored to different device types. Append the device type as a prefix to the component's name.

Example Directory Paths:
For a web version: /Card/WebCard
For a mobile version: /Card/MobileCard
This structure ensures a clear and organized system for component storage and retrieval.

File: frontend/src/App.css (hunk 3 above)
Comment (I2evard) [reply frontend/src/App.css:45]: Je sais pas si c'est une bonne approche tho, mais jfeel que je fais toujours des display: flex partout. Y'a pt pour les tags <></> que ca peut fuck so je pense pt rajouter un display-direction: column dans le futur si je vois que ca cause trop de problèmes

File: frontend/src/components/README.md (hunk 5 above)
Comment (uwponcel) [reply frontend/src/components/README.md:7]: Thats also gpt ;)

**Pull request merged**